src/Store/KeyBindingsStoreConnector.re

```diff
@@ -179,6 +179,16 @@ let start = () => {
         command: "workbench.action.files.save",
         condition: "editorTextFocus" |> WhenExpr.parse,
       },
+      {
+        key: "<TAB>",
+        command: "input.indent",
```

Comment:
`input` would refer to the text input component I think. VSCode has an`editor.action.indentLines` command, but no corresponding command for the reverse. But perhaps we could just invent that ourselves. Ideally this would work in both normal and visual modes though.